src/features/search/components/SearchBox.tsx

```diff
@@ -247,7 +243,7 @@ const SearchInputA11yContainer = styled.View({
   alignItems: 'center',
 })
 
-const StyledTouchableOpacity = styled(TouchableOpacity)({
+const StyledView = styled(View)({
```

Comment:
Ne pas importer View de react native mais faire ça :
```suggestion
const StyledView = styled.View({
```